docs/info-arch/doc-types/tutorial.md

```diff
@@ -0,0 +1,23 @@
+---
```

Comment:
I can't seem to find or visualize the pages that relate to `docs/info-arch/doc-types/*`. In my opinion, maybe they aren't needed, `docs/info-arch/ia.md` can explain the types, and if they have several sections then the sections should be self-explanatory on the sidebar. If we create an index per Diataxis section, I feel they will be missing entries in the long run.

Reply:
ok, I misunderstood where to find them, they are linked from  the headers of docs/info-arch/ia.md.
I'm not sure it's easily discoverable that way. I would prefer a link after the section header.